### DISCUSSION TITLE
Can't resolve '@babel/runtime/helpers/esm/regeneratorRuntime'

### DISCUSSION THREAD

**Discussion by kimmy-wang** [pedronauck/docz#1741] [Q&A]
Generating development SSR bundle failed

Can't resolve '@babel/runtime/helpers/esm/regeneratorRuntime' in '/Users/ying/work/seadaka-design-next/node_modules/antd/es/notification'

If you're trying to use a package make sure that '@babel/runtime/helpers/esm/regeneratorRuntime' is installed. If you're trying to use a local file make sure that the path is correct.

File: ../../../node_modules/antd/es/notification/index.js

**Comment by rodry86rmb** [comment]
si se puede solo kitar corchetes de blokeo
